pkg/cmd/cli/backup/create.go

```diff
@@ -154,13 +172,18 @@ func (o *CreateOptions) Complete(args []string, f client.Factory) error {
 }
 
 func (o *CreateOptions) Run(c *cobra.Command, f client.Factory) error {
-	backup := &api.Backup{
-		ObjectMeta: metav1.ObjectMeta{
-			Namespace: f.Namespace(),
-			Name:      o.Name,
-			Labels:    o.Labels.Data(),
-		},
-		Spec: api.BackupSpec{
+	var backup *api.Backup
+	backupBuilder := builder.ForBackup(f.Namespace(), o.Name).ObjectMeta(builder.WithLabelsMap(o.Labels.Data()))
+
+	if o.FromSchedule != "" {
```

Comment:
thinking of extracting this to a separate function so we can add a test for this, wdyt?